code/VersionedGridFieldOrderableRows.php

```diff
@@ -85,6 +90,10 @@ public function getSortTable(DataList $list)
         throw new Exception("Couldn't find the sort field '$field'");
     }
 
+    /**
+     * @param $grid
+     * @return array
+     */
     public function getURLHandlers($grid)

```

Comment:
Can we use `@inheritdoc` here instead of rewriting the doc? If not, it should match the signature of the interface. Same goes for multiple methods in this class.